packages/context-entries-link-traversal/lib/Keys.ts

```diff
@@ -18,4 +18,11 @@ export const KeysRdfResolveHypermediaLinks = {
   annotateSources: new ActionContextKey<AnnotateSourcesType>(
     '@comunica/bus-rdf-resolve-hypermedia-links:annotateSources',
   ),
+  /**
+   * A flag to indicate if relationships should strictly correspond to the current document's URL.
+   * Default true.
+   * Note: This flag is only effective in the context of the traversal of documents following the TREE specification.
+   */
+  treeSpecTraversalStrictMode:
```

Comment:
Instead of putting this inside of `KeysRdfResolveHypermediaLinks`, can we add a new `const KeysExtractLinksExtractTree` and put it in there?